about.html

```diff
@@ -12,52 +12,69 @@
 </head>
 
 <body>
+  <header>
+    <section class="header-container">
+        <div class="header-top">
+            <img src="img/lambda-black.png" class="header-image" alt="Lambda title">
```

Comment:
This image can do inside the nav and it should put the image and the anchors together like in the design file.